### PR TITLE
Feature/document create comment

### PR DESCRIPTION
**Pull request by atxmemo**
@msgerbush Review please, once we figure out how to correctly handle comment creation errors we can quickly merge this (if you think it looks good)

### CODE REVIEW

File: docs/apis/helpdesk.md [hunk 1]
```diff
@@ -295,3 +295,29 @@ card.services('helpdesk').on('showTicket', handler)
...
+-----|------|--------------
+`ticket_id`|`integer`| The `id` of the ticket this comment will be appended to
+`attributes`|`object`| See below for detailed requirements
+
+##### Attributes
+
+Name | Type | Description
+-----|------|--------------
+`body`|`string`| **Required**. The content of the comment.
+`created_at: datetime`|`string`| Created at date of the comment.  Must be a timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ`.
```
Comment (msgerbush) [inline docs/apis/helpdesk.md]: we shouldn't let them pass in the `created_at` datetime, we'll set that automatically when the record is created (if it's valid).

Comment (msgerbush) [inline docs/apis/helpdesk.md]: probably, we'll want to let them also set `is_public` so they can create hidden notes.  although, i think from the JS API, we should just use `public` which is consistent with our other APIs.

**Comment by babbtx** [comment]
Looks good. Want to merge it now or when we push the build?

**Comment by msgerbush** [comment]
I'm about to push a build, talking to Jeremy now to see if we can get it on the CDN.  So one second.

**Pull request merged**